gno.land/pkg/integration/testdata/restart.txtar

```diff
@@ -4,12 +4,12 @@
 loadpkg gno.land/r/demo/counter $WORK
 gnoland start
 
-gnokey maketx call -pkgpath gno.land/r/demo/counter -func Incr -gas-fee 1000000ugnot -gas-wanted 150000 -broadcast -chainid tendermint_test test1
+gnokey maketx call -pkgpath gno.land/r/demo/counter -func Incr -gas-fee 1000000ugnot -gas-wanted 200000 -broadcast -chainid tendermint_test test1
```

Comment:
Ah yes the random gas bumps nobody can predict :)

Reply:
It's honestly weird, and it didn't take too much time to check why it's increasing...